engine/src/flutter/impeller/geometry/path_component.cc

```diff
@@ -190,9 +190,9 @@ static inline Scalar ConicSolve(Scalar t,
                                 Scalar p2,
                                 Scalar w) {
   auto u = (1 - t);
-  auto coefficient_p0 = t * t;
+  auto coefficient_p0 = u * u;
```

Comment:
This was the bug you mentioned earlier this week? or something else ?

Reply:
Something else. Just an Oops. I don't think we've used this method before.

Reply:
👍